apps/earn-protocol-landing-page/app/page.tsx

```
@@ -125,7 +127,25 @@ export default async function HomePage() {
+      arksList: vault.arks,
+    }),
+  )
+
+  const interestRatesResults = await Promise.all(interestRatesPromises)
+
+  const vaultsDecorated = decorateCustomVaultFields({
+    vaults,
+    systemConfig: config,
+    decorators: {
+      arkInterestRatesMap: interestRatesResults.reduce((acc, curr) => {
+        return { ...acc, ...curr }
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)